### ISSUE TITLE
Plan for adding support for additional models like Llama

### ISSUE THREAD

**Issue by satreja1** [Pan-ML/panml#62]
Just curious if you've a plan in place/timeline to expand support for other models from the HuggingFace hub

**Comment by wanoz** [comment]
Yeah definitely planned to add them in. Will look into this over the next week. Happy to also work together if you want to contribute adding more of them as well.
